pyart/io/__init__.py

```diff
@@ -78,6 +78,7 @@
 from .uf import read_uf
 from .uf_write import write_uf
 from .grid_io import read_grid, write_grid, read_legacy_grid
+from .output_to_geotiff import write_grid_geotiff
```

Comment:
Add the `write_grid_geotiff` function in one of the documentation sections at the top of this file.  That way the function is included in the User Guide